base-isa.md

```diff
@@ -22,48 +23,45 @@ The highest two bits of the first byte are a format marker:
 | First byte    | Second Byte  | Comment                                  |
 |:--------------|:-------------|:-----------------------------------------|
 | `00 01 CCCC`  | `RRR RRR 00` | 2 register computation                   |
-| `00 SS CCCC`  | `RRR RRR ??` | when `SS != 01`, reserved for extensions |
-| `00 01 CCCC`  | `RRR RRR ??` | when `?? != 00`, reserved for extensions |
-| `00 01 11??`  | `RRR RRR ??` | reserved for extensions                  |
+| `00 01 11??`  | `???? ????`  | reserved for extensions                  |
+| `00 SS CCCC`  | `RRR RRR MM` | when `MM != 00`, reserved for extensions |
 | `01 01 CCCC`  | `RRR IIIII`  | immediate and 1 register computation     |
-| `01 SS CCCC`  | `RRR IIIII`  | when `SS != 01`, reserved for extensions |
+| `0? SS CCCC`  | `???? ????`  | when `SS != 01`, reserved for extensions |
 | `10 0 D CCCC` | `DDDDDDDD`   | (conditional) relative jump instruction  |
 | `10 1 ?????`  | `?????????`  | reserved for extension                   |
 | `11 ??????`   | `?????????`  | reserved for extensions                  |
 
-| Symbol | Meaning                      |
-|--------|------------------------------|
-| C      | opcode bit                   |
-| R      | register id                  |
-| I      | immediate                    |
-| D      | displacement                 |
-| S      | size (res.d)                 |
-| ?      | reserved for unknown purpose |
+| Symbol | Meaning         |
+|--------|-----------------|
+| C      | Opcode          |
+| R      | Register Id     |
+| I      | Immediate       |
+| D      | Displacement    |
+| S      | Operation Size  |
+| M      | Memory Mode     |
+| ?      | Arbitrary Value |
 
-### Execution of Reserved Instructions
-
-Execution of a reserved instruction is undefined behavior. A future extension will change this to "the CPU must trigger a specific interrupt."
+### Execution of Illegal and Reserved Instructions
 
+Execution of an _illegal_ or _reserved_ instruction is _unspecified_ behavior. A future extension will change this to require an interrupt to occur.
```

Comment:
This is not consistent with the text of the README. While it quite possibly makes sense in the context of other extensions being present, remember most people will read the README and then immediately read this document. If they contradict each other it would be quite confusing.

I also think lines like this should probably link to the relevant extension if it already exists. In most cases, at this point, it does.

Reply:
How is this inconsistent?